aws/cloudformation/drone-stack.yml

```diff
@@ -254,6 +255,19 @@ Resources:
             -
               Name: DRONE_SERVER_PRIVATE_MODE
               Value: true
+            # Store Drone Build logs in S3 instead of in a blob column in the logs table in the Drone Server relational database.
+            -
+              Name: AWS_DEFAULT_REGION
+              Value: us-east-1
+            -
+              Name: AWS_REGION
+              Value: us-east-1
+            -
+              Name: DRONE_S3_BUCKET
+              Value: code-dot-org
```

Comment:
awesome work getting this up and running, @sureshc ! Where in S3 are the logs stored? I don't see a bucket with this name.

Reply:
This bucket is in the Dev AWS Account in us-east-1

Reply:
Ahh, thanks! Is the idea that a developer should be able to find the log entry corresponding to their unit test run if it fails to appear in the UI, or is it rather that we think switching to S3 will fix the truncated log problem?

Reply:
The logs stored on S3 are in a drone-specific json format. They could probably be piped through some command-line json-parser filtering to make them more human-readable, but I assume the point is more to fix the truncated-log issue (without knowing the cause of the issue I'm not totally sure).